core/modules/wsched/BlendScheduler.cc

```diff
@@ -137,6 +137,134 @@ void BlendScheduler::_sortScanSchedulers() {
 }
 
 
+void BlendScheduler::queCmd(util::Command::Ptr const& cmd) {
+    std::vector<util::Command::Ptr> vect;
+    vect.push_back(cmd);
+    queCmd(vect);
+}
+
+
+/// &&& Change this to accept a vector of util::Command::Ptr const&
+void BlendScheduler::queCmd(std::vector<util::Command::Ptr> const& cmds) {
+    // Do the book keeping and determine which queue the tasks in 'cmds'
+    // should be added to. All tasks in 'cmds' must belong to the same
+    // user query and will go to the same scheduler.
+    LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd a");
```

Comment:
Please, consider using the predefined compiler-time constant `__func__` here in everywhere:
```
LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::" << __func__ << " a");
```